endpoints/openrtb2/auction.go

```diff
@@ -666,28 +666,27 @@ func validateNativeContextTypes(cType native1.ContextType, cSubtype native1.Cont
 	if cSubtype == 0 {
 		return nil
 	}
-
-	if cSubtype >= 500 {
-		return fmt.Errorf("request.imp[%d].native.request.contextsubtype can't be greater than or equal to 500. See https://iabtechlab.com/wp-content/uploads/2016/07/OpenRTB-Native-Ads-Specification-Final-1.2.pdf#page=39", impIndex)
-	}
```

Comment:
I'm a little nervous this was explicitly forbidden before. I also have no idea why this was enforced when the spec allows for it. Thoughts?

Reply:
Perhaps. I don't know why we wouldn't support exchange-specific type since we're just passing through data. The validation only exists to eliminate obviously wrong native responses.

Reply:
I am not aware of any specific reasons for not allowing exchange specific types either